packages/cdk/lib/constructs/engines/miniwdl/miniwdl-engine.ts

```diff
@@ -47,50 +44,9 @@ export class MiniWdlEngine extends Engine {
           MINIWDL__AWS__TASK_QUEUE: workerBatch.jobQueue.jobQueueArn,
           MINIWDL_S3_OUTPUT_URI: rootDirS3Uri,
         },
-        volumes: [this.toVolume(fileSystem, accessPoint)],
-        mountPoints: [this.toMountPoint("/mnt/efs")],
+        volumes: [this.toVolume(fileSystem, accessPoint, this.volumeName)],
+        mountPoints: [this.toMountPoint("/mnt/efs", this.volumeName)],
```

Comment:
Have you tested this against MiniWdl?

Reply:
yep. tried the words one and saw a successful run and saw outputs in s3